packages/mobile/src/identity/saga.ts

```diff
@@ -54,12 +53,11 @@ export function* validateRecipientAddressSaga({
     yield put(validateRecipientAddressSuccess(e164PhoneNumber, validatedAddress))
   } catch (error) {
     Logger.error(TAG, 'validateRecipientAddressSaga/Address validation error: ', error)
-    if (error.message in ErrorMessages) {
+    if (Object.values(ErrorMessages).includes(error.message)) {
```

Comment:
is this a stylistic preference, or different behavior?

Reply:
I believe it's different behavior. Correct me if I am wrong but I think the original checks property names in `ErrorMessages` whereas the latter checks values, which is what I actually want here.

I could also change the error messages I am throwing to be the properties if we want to stay consistent, but this was less work.

Reply:
To clarify, I wrote the original and noticed it wasn't working when adding on in this PR.

Reply:
Ah makes sense. Yeah my intuition was that it was different behavior, but just checking that it meant the previous behavior was broken 😄